lms/djangoapps/certificates/signals.py

```diff
@@ -68,16 +68,21 @@ def _listen_for_track_change(sender, user, **kwargs):  # pylint: disable=unused-
 
     user_enrollments = CourseEnrollment.enrollments_for_user(user=user)
     grade_factory = CourseGradeFactory()
+    expected_verification_status, _ = SoftwareSecurePhotoVerification.user_status(user)
     for enrollment in user_enrollments:
         if grade_factory.read(user=user, course=enrollment.course_overview).passed:
-            if fire_ungenerated_certificate_task(user, enrollment.course_id):
-                log.info(u'Certificate generation task initiated for {user} : {course} via track change'.format(
+            if fire_ungenerated_certificate_task(user, enrollment.course_id, expected_verification_status):
+                message = (u'Certificate generation task initiated for {user} : {course} via track change ' +
+                           u'with verification status of {status}'
+                           )
```

Comment:
Two minor style things:
- The opening paren holding the message should be on a line by itself, with the string on the next line and indented one tab-width to the right of `message`.
- The closing paren should be aligned directly under the 'm' in `message`.